app/server/routes/api.ts

```diff
@@ -227,6 +231,12 @@ router.get("/known-issues", async (_, response) => {
 
 router.post("/contact-us", contactUsFormHandler);
 
-router.post("/reminder", reminderHandler);
+router.post("/reminders", createReminderHandler);
+router.get(
+  "/reminders/status",
+  membersDataApiHandler("user-attributes/me/reminders", "MDA_REMINDERS_STATUS")
```

Comment:
Why does this endpoint get routed through `membersDataApiHandler` but not the other two?
Using `membersDataApiHandler` will give you loads for free, like credentials and logging .

Reply:
Yeah this is slightly confusing. This is the only request that actually goes to members data api - the others go direct to our reminders endpoint. @tomrf1  I can't remember now why we decided this one had to go through mdapi?

Reply:
I might be missing something here, but this is the only new request to MDAPI so it's the only one that should be using `membersDataApiHandler`.

Reply:
Yeah that's right. I was meaning more architecturally, why we decided this endpoint should be implemented in MDAPI?

Reply:
Ah right.
This request reveals some information about a user (have they signed up?), so it needs to be auth'd, and MDAPI is the right place for that.
The cancel/reactivate endpoints do not tell the client anything, so no auth required

Reply:
Gotcha! Makes sense 👍

Reply:
The cancel/reactivate endpoints are revealing anything about the user but they still make changes to a user's settings. For this reason I believe they should also require authentication.
Otherwise these endpoints will allow people to change each other's reminder settings. Are we ok with that risk?

There might be perfectly valid reasons for doing it this way that I'm just not aware of though. In which case I would like to see them documented in this PR so we can better understand our choices in the future.

Reply:
These endpoints now take a `reminderCode` rather than the email address. This code is given to braze by our reminders system. It's not authentication, but it's good enough for this purpose

Reply:
Yeah just to expand on this - `reminderCode` is a `uuid` that we generate at the point of signup. The only way to get access to this code is through the MDAPI call, or the email that we send to a user (it'll be in the query string of the unsub link). Like Tom said, it's not really an authentication system - but it does prevent altering other users reminders. Do you think that is secure enough?

Reply:
Sorry for the delay in replying. Yeah, I think that's a great improvement 👍